include/Hcal/HcalRecProducer.h

```diff
@@ -89,14 +101,37 @@ class HcalRecProducer : public framework::Producer {
   /// Voltage by average MIP
   double voltage_per_mip_;
 
-  /// Gain [mv/ADC]
-  double gain_;
-
-  /// Pedestal [ADC units]
-  double pedestal_;
-
   /// Strip attenuation length [m]
   double attlength_;
+
+  /// Pulse function and correction graphs
+  mutable TF1 pulseFunc_;
+  mutable TGraph correctionAmpl_;
+  mutable TGraph correctionTOA_;
```

Comment:
This is a perfect spot to put some documentation about how these corrections are calculated.

Using a multiline-doxygen comment, you'll have plenty of room and it will show up in the docs.

```cpp
/**
 * Correction to the pulse amplitude.
 * This was calculated by...
 */
```

Reply:
Cool, thanks Tom! Just updated the docs